### PR TITLE
daniel-db-nodes-on-cloud --> main

### PR DESCRIPTION
**Pull request by YoungMasterGandalf**

### CODE REVIEW

File: forloop_modules/function_handlers/database_handlers.py [hunk 1]
```diff
@@ -424,13 +416,18 @@ def execute(self, node_detail_form):
         # else:
         #     raise HTTPException(status_code=response.status_code, detail="Error requesting new node from api")
         
-    def direct_execute(self, db_name, dbtable_name, selected_columns, column_name, operator, value, limit, new_var_name):        
-        db_table = get_db_table_from_db(table_name=dbtable_name, db_name=db_name)
+    def direct_execute(self, db_name, db_table_name, select, where_column_name, where_operator, where_value, limit, new_var_name):
+        db_name = parse_comboentry_input(input_value=db_name)
+        db_table_name = parse_comboentry_input(input_value=db_table_name)
+        select = parse_comboentry_input(input_value=select)
+        where_column_name = parse_comboentry_input(input_value=where_column_name)
```
Comment (DovaX) [inline forloop_modules/function_handlers/database_handlers.py:423]: Why is it in direct execute - that sounds wrong

**Review by DovaX — CHANGES_REQUESTED** [review]

File: forloop_modules/function_handlers/database_handlers.py (hunk 1 above)
Comment (YoungMasterGandalf) [reply forloop_modules/function_handlers/database_handlers.py:423]: I will move it to `execute` @DovaX

Comment (DovaX) [reply forloop_modules/function_handlers/database_handlers.py:423]: Yes, please do it

**Review by DovaX — CHANGES_REQUESTED** [review]

File: forloop_modules/function_handlers/database_handlers.py (hunk 1 above)
Comment (YoungMasterGandalf) [reply forloop_modules/function_handlers/database_handlers.py]: Done @DovaX

**Review by DovaX — APPROVED** [review]

**Comment by DovaX** [comment]
merge it please

**Pull request merged**